### ISSUE TITLE
Cannot build docker compose using README instructions - wrong pnpm version

### ISSUE THREAD

**Issue by clausjs** [RaunoT/plex-rewind#97]
### Plex Rewind Version

latest: v1.8.7

### Description

Build installs pnpm@9.0.6 then fails because expecting pnpm@8.13.1 which is the version used to generate the lockfile

### Steps to reproduce

1. Pull down repo
2. Follow README instructions for docker compose, which includes project building
3. Observe error

### Screenshots

![plex-rewind-error](https://github.com/RaunoT/plex-rewind/assets/12068849/c0514142-d8c3-44fc-9cf4-e0c1026e5096)

### Logs

```shell
[+] Building 11.0s (13/21)                                                                               docker:default
 => [plex-rewind internal] load build definition from Dockerfile                                                   0.0s
 => => transferring dockerfile: 2.17kB                                                                             0.0s
 => [plex-rewind internal] load metadata for docker.io/library/node:18-alpine                                      0.8s
 => [plex-rewind internal] load .dockerignore                                                                      0.0s
 => => transferring context: 120B                                                                                  0.0s
 => [plex-rewind internal] load build context                                                                      0.1s
 => => transferring context: 497.86kB                                                                              0.1s
 => [plex-rewind base 1/2] FROM docker.io/library/node:18-alpine@sha256:4837c2ac8998cf172f5892fb45f229c328e4824c4  4.4s
 => => resolve docker.io/library/node:18-alpine@sha256:4837c2ac8998cf172f5892fb45f229c328e4824c43c8506f8ba9c7996d  0.0s
 => => sha256:4837c2ac8998cf172f5892fb45f229c328e4824c43c8506f8ba9c7996d702430 1.43kB / 1.43kB                     0.0s
 => => sha256:267ef0e247ce1ace38e92d80c5c0ebb0aa2ce34e8ba28998f8f1acd886451e49 1.16kB / 1.16kB                     0.0s
 => => sha256:1835bef2bac85b1699df958a9ee9d867ec68551e192317c0a36f5e8d8987280b 7.21kB / 7.21kB                     0.0s
 => => sha256:4abcf20661432fb2d719aaf90656f55c287f8ca915dc1c92ec14ff61e67fbaf8 3.41MB / 3.41MB                     0.5s
 => => sha256:45a0166cf96b2a4f328191f78f73e68e0e340450a962ff6fc34013111c014d26 39.82MB / 39.82MB                   2.0s
 => => sha256:832e0dc1fe41d061d47d41e00abf6a9dab0c399d69bae854ef1bffe1976c2df0 1.38MB / 1.38MB                     0.2s
 => => sha256:8ae971f79f99381da4a83f2cb63aa502fb847cc81a2f270326753f6289562dfc 454B / 454B                         0.2s
 => => extracting sha256:4abcf20661432fb2d719aaf90656f55c287f8ca915dc1c92ec14ff61e67fbaf8                          0.3s
 => => extracting sha256:45a0166cf96b2a4f328191f78f73e68e0e340450a962ff6fc34013111c014d26                          2.1s
 => => extracting sha256:832e0dc1fe41d061d47d41e00abf6a9dab0c399d69bae854ef1bffe1976c2df0                          0.1s
 => => extracting sha256:8ae971f79f99381da4a83f2cb63aa502fb847cc81a2f270326753f6289562dfc                          0.0s
 => [plex-rewind base 2/2] RUN npm install -g pnpm                                                                 2.9s
 => [plex-rewind deps 1/4] RUN apk add --no-cache libc6-compat                                                     1.3s
 => [plex-rewind runner 1/8] WORKDIR /app                                                                          0.0s
 => [plex-rewind runner 2/8] RUN addgroup --system --gid 1001 nodejs                                               0.5s
 => [plex-rewind runner 3/8] RUN adduser --system --uid 1001 nextjs                                                0.5s
 => [plex-rewind deps 2/4] WORKDIR /app                                                                            0.0s
 => [plex-rewind deps 3/4] COPY package.json yarn.lock* package-lock.json* pnpm-lock.yaml* ./                      0.0s
 => ERROR [plex-rewind deps 4/4] RUN   if [ -f yarn.lock ]; then yarn --frozen-lockfile;   elif [ -f package-lock  1.5s
------
 > [plex-rewind deps 4/4] RUN   if [ -f yarn.lock ]; then yarn --frozen-lockfile;   elif [ -f package-lock.json ]; then npm ci;   elif [ -f pnpm-lock.yaml ]; then pnpm i --frozen-lockfile;   else echo "Lockfile not found." && exit 1;   fi:
1.455  ERR_PNPM_BAD_PM_VERSION  This project is configured to use v8.13.1 of pnpm. Your current pnpm is v9.0.6
1.455
1.455 If you want to bypass this version check, you can set the "package-manager-strict" configuration to "false" or set the "COREPACK_ENABLE_STRICT" environment variable to "0"
```

### Platform

desktop

### Device

Windows PC

### Operating System

Windows 10

### Browser and version

N/A

**Comment by clausjs** [comment]
The package.json appears to point to the correct version of pnpm (8.13.1) but if I run the build it errors mentioning 9.0.6. I do not have pnpm installed globally or for any other projects on the machine I'm attempting to run this on.

Running `npx pnpm -v` outputs: `8.13.1`

**Comment by RaunoT** [comment]
Hello @clausjs 👋

I've just published a `2.0.0` version of `plex-rewind`. 
Since this issue deals with parts of the code that have been completely refactored, I will close this issue for now and suggest you attempt updating.

If you still run into any issues, feel free to open a new ticket.
